docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,128 @@
+the relevant cell.
+
+This noise type is called :code:`leave_blank` in the configuration. It takes one parameter:
+
+.. list-table:: Parameters to the :code:`leave_blank` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that a cell in the column being configured is blank.
+      This probability is the same for all cells in a column;
+      missingness is not correlated with any attributes.
+    - 0.01 (1%)
+
 Choose the wrong option
 -----------------------
 
-Write the wrong zipcode digits
-------------------------------
+Some survey questions or administrative forms ask the respondent to choose
+between a list of options.
+This noise type occurs when the wrong option is chosen by mistake.
+The options available depend on the column; see the :ref:`datasets page <datasets_main>` for
+lists of options that correspond to each column.
+All wrong options are equally likely.
+
+This noise type is called :code:`choose_wrong_option` in the configuration.
+It takes one parameter:
+
+.. list-table:: Parameters to the :code:`choose_wrong_option` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that, for a cell in the column being configured, the wrong option is chosen.
+      The probability of this noise type is the same for all cells in a column;
+      it is not correlated with any attributes.
+    - 0.01 (1%)
+
+Write the wrong ZIP code digits
+-------------------------------
+
+When reporting a ZIP code on a survey or form, people may misremember or misreport
+the digits.
+They are probably more likely to get confused about the last few digits (which identify
+the small, specific area) than the first few (which will be the same over a larger area).
+
+This noise type is called :code:`write_wrong_zipcode_digits` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the :code:`write_wrong_zipcode_digits` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability of a cell being *considered* to have this noise type.
+      One way to think about this is the probability that a ZIP code is reported by someone who isn't sure of their ZIP code.
+      This probability is the same for all cells in a column; it is not correlated with any attributes.
+      Whether or not there are actually any errors depends on the next parameter.
+    - 0.01 (1%)
+  * - :code:`digit_probabilities`
+    - A list of probabilities, one for each digit in a (5-digit) ZIP code.
+      The first item in this list is the probability **in a cell considered for this noise type** that the first digit of the ZIP
+      code will be wrong, the second item is the same probability for the second digit, and so on.
+    - [0.2, 0.2, 0.2, 0.36, 0.36]
```

Comment:
How about:

A list of probabilities, one for each digit in a (5-digit) ZIP code. The first value in this list is the conditional probability--given that a cell has been chosen for this noise type--that the first digit of the ZIP code in this cell will be wrong. The second value is the conditional probability that the second digit in the chosen cell will be wrong, and so on.

Reply:
I like most of this but "conditional probability" doesn't strike me as approachable. I'll try to rework that phrase.